pkg/datapath/linux/probes/probes.go

```diff
@@ -164,39 +163,6 @@ func (*ProbeManager) Probe() Features {
 	return features
 }
 
-func (p *ProbeManager) probeSystemKernelHz() (int, error) {
-	out, err := exec.WithTimeout(
-		defaults.ExecTimeout,
-		"cilium-probe-kernel-hz",
-	).Output(log, false)
-	if err != nil {
-		return 0, fmt.Errorf("Cannot probe CONFIG_HZ")
-	}
-	hz := 0
-	warp := 0
-	n, _ := fmt.Sscanf(string(out), "%d, %d\n", &hz, &warp)
-	if n == 2 && hz > 0 && hz < 100000 {
-		return hz, nil
-	}
-	return 0, fmt.Errorf("Invalid probed CONFIG_HZ value")
-}
-
-// SystemKernelHz returns the HZ value that the kernel has been configured with.
-func (p *ProbeManager) SystemKernelHz() (int, error) {
-	config := p.features.SystemConfig
-	if config.ConfigKernelHz == "" {
-		return p.probeSystemKernelHz()
-	}
```

Comment:
If the config is available, should we keep that part in tact? Here only the fallback does the probing.

Other than that lgtm.